portal/guides/nft-gated-website.mdx

```diff
@@ -1,134 +1,270 @@
-2. Create new project (if you don't have one).
-3. Add a module: _Bundle Collection (ERC 1155)_.
-4. Add a name and description for your collection. (This will be displayed when users view the collection page on Opensea).
-5. Click Deploy → confirm both steps in your wallet.
-6. Click **Mint** on the top right corner to start minting your Membership NFT.
-7. Add a name and description for your individual NFT. (This will be displayed when users view the individual NFT page on OpenSea).
-8. You can mint multiple NFT's which represent different tiers of membership.
-9. Your NFTs have now been minted, and after a few minutes will appear on Opensea under your collection.
+In this guide, we will create a website that restricts content based on owning an NFT using React.
 
-![thirdweb collection dashboard with 3 tiers of membership NFTs. (blue, red, green).example on OpenSea: https://opensea.io/collection/membership-1155](/assets/portal/guides/nft-gated-website/gated-nft.png)
-thirdweb collection dashboard with 3 tiers of membership NFTs. (blue, red, green).example on OpenSea: [https://opensea.io/collection/membership-1155](https://opensea.io/collection/membership-1155)
+You can grab a copy of the final project from our examples repository: [https://github.com/thirdweb-example/NFT-Gated-Website](https://github.com/thirdweb-example/NFT-Gated-Website).
 
-**[0x4465...65EB]** is your NFT contract address.
+## Mint an NFT Drop
 
-## Let's write some code
+To create an NFT gated website, we will need either an [NFT Collection](/contracts/nft-collection), [NFT Drop](/contracts/nft-drop), [Edition](/contracts/edition), or an [Edition Drop](/contracts/edition-drop) contract already deployed on a blockchain. If you don’t have one created, you can create one using thirdweb’s [TypeScript SDK](https://docs.thirdweb.com/typescript/sdk.nftdrop) or thirdweb’s [dashboard](https://thirdweb.com/dashboard).
 
-We're going to setup a **private** section of our website for our **NFT holders**. In this guide, we'll be using **React** and the Javascript SDK.
+<MdxAlert status="warning">
+If your NFTs are an NFT Drop or Edition Drop contract, set up claim phases before using the claiming functions mentioned later in the guide.
+</MdxAlert>
 
-### 1. Install the SDK in your web project
+For our example, I am using an NFT Drop contract called Cookie Club for members of the Cookie Club. 🤫
 
-```bash
-npm install @3rdweb/sdk ethers
-```
+Link to NFT Drop dashboard for Cookie Club: [https://thirdweb.com/dashboard/rinkeby/nft-drop/0x66463b3C1EBf08b9dE889BCc0A5cbf29dc0e2B7a?tabIndex=0](https://thirdweb.com/dashboard/rinkeby/nft-drop/0x66463b3C1EBf08b9dE889BCc0A5cbf29dc0e2B7a?tabIndex=0)
+
+## Clone the template repository
+
+To get started will use a `cra-javascript-template` which already includes a working SDK setup. This template will make it easy to learn and save time when getting started with our project.
+
+<MdxAlert status="info">
```

Comment:
if you mention this you can also add a snippet of index.js, it's short and it will give all the tools for people to run it themselves

Reply:
I know, but it's very easy to add index.js here, it's just like adding 10 lines and gives more value.

Template is the easier way, but people are free to not use it